lib/logflare_web/live/backends/backends_live.ex

```diff
@@ -289,4 +263,33 @@ defmodule LogflareWeb.BackendsLive do
     |> assign(:backend, backend)
     |> assign(:form_type, Atom.to_string(backend.type))
   end
+
+  defp parse_metadata(data) when is_binary(data) do
+    data
+    |> String.split(",")
+    |> Enum.reduce(%{}, fn pair, acc ->
+      case String.split(pair, "=", parts: 2) do
+        [key, value] -> Map.put(acc, String.trim(key), String.trim(value))
+        _ -> acc
+      end
+    end)
+  end
+
+  defp changeset_to_flash_message(changeset) do
```

Comment:
Would be good to extract this out to it's own utils function as this is used in quite a lot of places.

Reply:
Done in https://github.com/Logflare/logflare/pull/2680 :pray: